src/app/config/config.service.ts

```diff
@@ -20,7 +20,7 @@ import { Injectable } from '@angular/core';
 @Injectable()
 export class ConfigService {
 
-  version = '2.0.0';
+  version = '2.0.14';
```

Comment:
This is our project version, right? If we bump it here, we should probably bump the version in `package.json` too - or in the first place only save a version in one file. Opening an issue about this, as it has nothing to do with this PR: #73